blocks/avm-report/avm-report.css

```diff
@@ -0,0 +1,96 @@
+.avm-report.block {
+  padding: 2em 0;
+  max-width: 600px;
+}
+
+.avm-report.block h2 {
+  font-size: var(--heading-font-size-m);
+  margin-bottom: .5em;
+}
+
+.avm-report.block p {
+  letter-spacing: var(--letter-spacing-s);
+}
+
+.avm-report.block form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2em;
+  justify-content: center;
+  align-items: center;
+}
+
+.avm-report.block form .avm-input {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.avm-report.block form .avm-input input {
+  display: block;
+  padding-left: 10px;
+  margin-right: 16px;
+  height: 50px;
+  border: 1px solid var(--border-medium-gray);
+}
+
+.avm-report.block form .avm-input input[name="avmaddress"] {
+  margin-right: 16px;
+  width: 250px;
+}
+
+.avm-report.block form .avm-input input[name="avmunit"] {
+  padding-left: 10px;
+  width: 67px;
+}
+
+.avm-report.block form button[type="submit"] {
```

Comment:
I would move button[type="submit"] to the main styles. I've seen pretty the same styles on pages, I used to work. 
The missed part is changing the color text and background on hover. 
button[type="submit"]:hover {
    background-color: var(--white);
    color: var(--border-medium-grey);
}